main.js

```diff
@@ -77,6 +99,25 @@ function decode({hashlink}) {
  *
  * @returns {Promise<boolean>} true if the hashlink is valid, false otherwise.
  */
-async function verify({hashlink, data, resolvers}) {
+async function verify(options) {
+  let hashlink, data, resolvers;
+
+  if(!options) {
+    throw new Error('Hashlink and/or data params are required.');
+  }
+
+  if(typeof options === 'string') { // Convenience usage, `create(url)`
+    hashlink = options;
+  } else {
+    hashlink = options.hashlink;
+    data = options.data;
+    resolvers = options.resolvers;
+  }
+
+  if(!data) {
```

Comment:
We need to add resolver support... in another PR. Would rather throw these sorts of errors when the URL is attempted to be resolved.

Reply:
The only reason I add it now is because it throws a really dense / not-understandable error now, if you don't include `data` (something about a missing `slice` etc).